vars.py

```diff
@@ -1,98 +1,107 @@
+            # use the original Soundtrack instead of arranged
+            self.legacySoundtrack   = config_vars['legacySoundtrack']
+            self.doNotSkipCutscenes = config_vars['doNotSkipCutscenes']
+
+            # ----Accessibility for blind
+            self.skip_cutscene_flag = config_vars['skip_cutscene_flag']
+            self.skip_diag_flag     = config_vars['skip_diag_flag']
+            self.play_TTS_flag      = config_vars['play_TTS_flag']
+            self.rails_trials       = config_vars['rails_trials']
+            self.rails_egg_hunt     = config_vars['rails_egg_hunt']
+
+            # ----Blitzball
+            self.blitzWinValue      = config_vars['blitzWinValue']  # No default value required
+            self.blitzOvertime      = config_vars['blitzOvertime']  # Set to False, no need to change ever.
+            self.blitzFirstShotVal  = config_vars['blitzFirstShotVal']
+            self.oblitzAttackVal    = config_vars['oblitzAttackVal']  # Used for RNG manip tracking
+
+            # ----Sphere grid
+            self.fullKilikMenu      = config_vars['fullKilikMenu']  # Default to False
+            self.earlyTidusGridVal  = config_vars['earlyTidusGridVal']  # Default False
+            self.earlyHasteVal      = config_vars['earlyHasteVal']  # Default -1
+            self.wakkaLateMenuVal   = config_vars['wakkaLateMenuVal']  # Default False
+            self.endGameVersionVal  = config_vars['endGameVersionVal']  # Default 0
+
+            # ----Equipment
+            self.zombieWeaponVal    = config_vars['zombieWeaponVal']  # Default 255
+            self.lStrikeCount       = config_vars['lStrikeCount']  # Default 0
+
+            # ----RNG Manip
+            self.yellows            = config_vars['yellows']  # ?
+            self.confirmedSeedNum   = config_vars['confirmedSeedNum']  # ?
+            self.skipZanLuck        = config_vars['skipZanLuck']  # ?
+
+            # ----Other
+            self.newGame            = config_vars['newGame']  # ?
+            self.selfDestruct       = config_vars['selfDestruct']  # Default False
+            self.YTKFarm            = config_vars['YTKFarm']  # Default to 0
+            self.rescueCount        = config_vars['rescueCount']  # Default to 0
+            self.fluxOverkillVar    = config_vars['fluxOverkillVar']  # Default to False
+            self.tryNEVal           = config_vars['tryNEVal']  # Based on
+            self.neArmorVal         = config_vars['neArmorVal']  # Default 255
+            self.neBattles          = config_vars['neBattles']  # Default to 0
+            # Decides in which zone we charge Rikku OD after reaching Zanarkand.
+            self.neaZone            = config_vars['neaZone']
+
+            # ----Nemesis variables, unused in any%
+            self.nemAPVal           = config_vars['nemAPVal']  # Default to 1
+            self.yojimboIndex       = config_vars['yojimboIndex']
+        except Exception as E:
```

Comment:
Odds are this only needs to be around line 11. And we either need to gracefully handle missing config or raise an exception here. Otherwise the moment the TAS gets after this point (with the exception), it'll crash when it tries to pull into out of here that was skipped because of this error handling

Reply:
I'll give it another look over and think a bit on how to solve the issue without mutliplying the code footprint.